Assets/AnkrSDK/Scripts/Core/Implementation/Contract.cs

```diff
@@ -141,9 +142,9 @@ private async UniTask LoadReceipt(string transactionHash, ITransactionEventHandl
 			}
 		}
 
-		private TransactionInput CreateTransactionInput(string methodName, object[] arguments)
+		private async UniTask<TransactionInput> CreateTransactionInput(string methodName, object[] arguments)
 		{
-			var activeSessionAccount = EthHandler.DefaultAccount;
+			var activeSessionAccount = await _ethHandler.GetDefaultAccount();
```

Comment:
can we pass the account as a parameter, and not make CreateTransactionInput an async?